src/main/kotlin/calculator/domain/Operator.kt

```diff
@@ -0,0 +1,16 @@
+package calculator.domain
+
+enum class Operator(
+    val symbol: String,
+    val operation: (left: Operand, right: Operand) -> Operand
+) {
+
+    PLUS("+", { left, right -> left + right }),
+    MINUS("-", { left, right -> left - right }),
+    TIMES("*", { left, right -> left * right }),
+    DIVIDE("/", { left, right -> left / right });
+
+    companion object {
+        fun of(symbol: String): Operator = values().find { it.symbol == symbol } ?: throw IllegalArgumentException("사칙 연산이 아닌 연산자는 지원하지 않습니다")
```

Comment:
```suggestion
        fun of(symbol: String): Operator = values().find { it.symbol == symbol }
            ?: throw IllegalArgumentException("사칙 연산이 아닌 연산자는 지원하지 않습니다")
```
엘비스 연산자를 활용해서 예외처리 잘 해주셨습니다.
개인적으로는 엘비스 연산자나 메서드 체이닝을 사용할 때 라인이 길어지면 줄바꿈을 한 번 해주는 게 읽기 좋더라구요 😁